### PR TITLE
[.github] Make CODEOWNERS paths more precise. Issue #13470.

### LINKED ISSUE #13470: Use absolute path in CODEOWNERS when referring to specific files
AFAICT, these files should be
https://github.com/lowRISC/opentitan/blob/d8af32ff61026d72a7586c66870feaa162335654/.github/CODEOWNERS#L29-L36

### PR DESCRIPTION
**Pull request by dmcardle**
Fixes #13470

Signed-off-by: Dan McArdle <dmcardle@google.com>

### CODE REVIEW

**Review by milesdai — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -54,10 +54,10 @@ util/topgen/        @msfschaffner @tjaychen @Jacob-Levy
...
 
 # DV related common files
-dv/                 @lowRISC/ot-dv-reviewers
-fpv/                @lowRISC/ot-dv-reviewers
-formal/             @lowRISC/ot-dv-reviewers
-# lint/             # TBD
+/hw/**/dv/              @lowRISC/ot-dv-reviewers
+/hw/**/fpv/             @lowRISC/ot-dv-reviewers
+/hw/**/formal           @lowRISC/ot-dv-reviewers
+# /hw/**/lint/                 # TBD
```
Comment (msfschaffner) [inline .github/CODEOWNERS:60]: do we have an `ot-dd-reviewers` for "digital design"? if yes, lint could be assigned to that.

Comment (dmcardle) [reply .github/CODEOWNERS:60]: Sadly, it does not appear we do. https://github.com/orgs/lowRISC/teams/ot-dd-reviewers is a 404.

Comment (msfschaffner) [reply .github/CODEOWNERS:60]: @moidx @mundaym @tjaychen should we add that list as well?

**Review by drewmacrae — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (msfschaffner) [reply .github/CODEOWNERS:60]: I would put down the usual suspects of RTL reviewers:
@eunchan 
@msfschaffner 
@tjaychen 
@vogelpi 
@GregAC

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -26,14 +26,14 @@ BUILD.bazel         @cfrantz @drewmacrae
...
-util/*gen/          @msfschaffner @tjaychen
-util/uvmdvgen*      @sriyerg
-util/regtool.py     @Jacob-Levy
-util/reggen/        @Jacob-Levy
-util/tlgen.py       @eunchan @msfschaffner @tjaychen
-util/tlgen/         @eunchan @msfschaffner @tjaychen
-util/topgen.py      @msfschaffner @tjaychen @Jacob-Levy
-util/topgen/        @msfschaffner @tjaychen @Jacob-Levy
+/util/*gen/          @msfschaffner @tjaychen
+/util/uvmdvgen/      @sriyerg
```
Comment (sriyerg) [inline .github/CODEOWNERS:30]: Could you add `@sriyerg` for `/util/dvsim` as well?

File: .github/CODEOWNERS (hunk 1 above)
Comment (sriyerg) [reply .github/CODEOWNERS:60]: Maybe also @lowRISC/ot-dv-reviewers for one DV rep, since DV code also goes through lint.

Comment (dmcardle) [reply .github/CODEOWNERS:65]: Following up on this, was the idea to uncomment line 65 and add `ot-dd-reviewers` once it's created? Should i add msfschaffner's list from two comments back in the meantime?

**Comment by alphan** [comment]
🏓

**Comment by msfschaffner** [comment]
Should this be revived and cleaned up or has this become irrelevant?
CC @mundaym

**Comment by moidx** [comment]
I think we probably need to start a new PR at this point given all the committer changes that have been made this year.

**Pull request closed without merging**